tox.ini

```diff
@@ -53,6 +53,7 @@ extras =
     test
 deps =
     xdist: pytest-xdist
+    cov: pytest-cov
```

Comment:
Why is this an additional dependency, it is one of the `[test]` dependencies.

Reply:
I figure that this makes it explicit, so it doesn't necessarily need `[test]`. I can remove it if that doesn't make sense